src/refract/VisitorUtils.h

```diff
@@ -125,6 +125,32 @@ namespace refract
         }
     };
 
+    struct GetEnumValue {
```

Comment:
Is there anyway to overload `GetValue` for enum element? I tried and failed, but maybe a trick I'm not aware of.